superset/assets/package.json

```diff
@@ -89,6 +89,7 @@
     "classnames": "^2.2.5",
     "d3-array": "^1.2.4",
     "d3-color": "^1.2.0",
+    "d3-format": "^1.3.2",
```

Comment:
I believe you should use this instead: https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-number-format

Reply:
fixed